linux/sound.cpp

```diff
@@ -210,9 +200,9 @@ int CSound::Init ( const int /* iNewPrefMonoBufferSize */ )
     // without a Jack server, Jamulus makes no sense to run, throw an error message
     if ( bJackWasShutDown )
     {
-        throw CGenErr ( tr ( "The Jack server was shut down. This software "
-                             "requires a Jack server to run. Try to restart the software to "
-                             "solve the issue." ) );
+        throw CGenErr ( tr ( "The JACK server was shut down but this software "
```

Comment:
The "but" isn't necessary. I think the original first two sentences are fine.

Or, alternatively with a semicolon: The JACK server was shut down; this software requires a JACK server to run.

Also, is "this software" referencing Jamulus? If so, should we just say "Jamulus" instead? Since we're dealing with a lot of interacting hardware and software, I think saying Jamulus directly clears up any possible confusion a user may have.

Reply:
BTW, when these messages come up, do they have some indication (in the window bar for example) that they are from Jamulus? If so, then that might make the writing of some messages (maybe not this one) a bit simpler.

Reply:
Not sure?

Reply:
![JACK issue](https://user-images.githubusercontent.com/20726856/119879389-79080600-bf2b-11eb-9d73-9c8582403027.png)

Just checked this error and it shows that it's from Jamulus